### ISSUE TITLE
Alignment issue with Task Lists

### ISSUE THREAD

**Issue by Morgul** [vsch/idea-multimarkdown#18]
I'm running into an alignment issue with Task Lists:

---

<img width="185" alt="screen shot 2015-09-04 at 5 28 38 pm" src="https://cloud.githubusercontent.com/assets/770655/9695838/f870f414-532b-11e5-9b59-e98c8e600e88.png">

---

I'm sure I could fix it with some custom CSS, but would rather report to error, as it doesn't bother me _too_ much. I'm wondering if this is an OS X issue; I haven't confirmed it on other platforms yet.

**Comment by Morgul** [comment]
Additionally, I just noticed the checkboxes can be checked. They should probably be read only...

**Comment by vsch** [comment]
Actually @Morgul, this how I wound up forking and releasing this plugin and working like a madman. I started with exactly the same line of thinking. I'll just modify the CSS. 

The problem is deeper and it is the reason why I tried the icon then image workaround for the input fields. The HTML rendering engine is the swing HTMLEditorKIt which has a slew of limitations in its implementation.

There was no way to disable the inputs so I tried to work-around this by using images. It does not implement list-style-image in a workable way either, so I had to chew up the HTML after markdown processing and insert `<img>` tags but then it causes other issues.

I did change the tasks to display as either square list items that don't show the check or an image. **I suspect that you have an old version of the plugin.** The later releases either use icons or empty squares. No inputs because they are not controllable at all. Cannot size them, move them, add margins, padding, etc.

I am working on changing to javafx WebView. I already have it running and it looks much better than HTMLEditorKit but I have to redo the stylesheets and try out all the features before I can say it is ready.

Additionally, with the WebView the plugin will require running IDEA on jdk 1.8. Not a problem for the latest releases but it will not work for anything earlier than build 141. I for one would welcome the cut-off. It is a pain trying to support half-dozen old builds of IDEA.

Most likely there will be one more release for old builds, then only major bugs. The rest of development will be on the new version under jdk 1.8.
